docs/source/release/v6.3.0/muon.rst

```diff
@@ -5,77 +5,67 @@ Muon Changes
 .. contents:: Table of Contents
    :local:
 
-.. warning:: **Developers:** Sort changes under appropriate heading
-    putting new features at the top of the section, followed by
-    improvements, followed by bug fixes.
-
 
 Frequency Domain Analysis
 -------------------------
 
 New Features
 ############
 
-- Can now select the unit for the frequency spectra (MHz or Gauss), by picking either `Frequency` or `Field` respectfully.
-
-Improvements
-############
+- Users can now select the unit for the frequency spectra (MHz or Gauss) in plotting, by picking either ``Frequency`` or ``Field`` respectively after a transform has been calculated.
 
 Bugfixes
 ########
 
 - Fixed a bug that prevented the frequency spectra from being plotted when data was binned.
 
-Muon Analysis
--------------
-
-New Features
-############
-
 
 Muon Analysis and Frequency Domain Analysis
 -------------------------------------------
 
 New Features
 ############
 
-- Instead of plotting the confidence interval of a fit as an error bar, it is now represented by a shaded region.
+- **Instead of plotting the confidence interval of a fit as an error bar, it is now represented by a shaded region.**
```

Comment:
is this supposed to be in bold?

Reply:
Any note that accompanies an image is supposed to be in bold.